src/coreclr/jit/importer.cpp

```diff
@@ -3654,10 +3654,11 @@ GenTree* Compiler::impIntrinsic(GenTree*                newobjThis,
                                 NamedIntrinsic*         pIntrinsicName,
                                 bool*                   isSpecialIntrinsic)
 {
-    bool           mustExpand  = false;
-    bool           isSpecial   = false;
-    bool           isIntrinsic = false;
-    NamedIntrinsic ni          = NI_Illegal;
+    bool mustExpand  = false;
+    bool isSpecial   = false;
+    bool isIntrinsic = false;
```

Comment:
```suggestion
    const bool isIntrinsic = (methodFlags & CORINFO_FLG_INTRINSIC) != 0;
```
Nit: To match the style of the other isIntrinsic local